BatchAuctionOptimization/batchauctions.tex

```diff
@@ -188,82 +189,63 @@ \section{Problem statement}
+a tuple~$ (j,k,\overline{x},\pi) $ that reads
 \vspace{-.6cm}
 \begin{center}
   \emph{
     "Buy (at most) $ \overline{x} $ units of token $ \tau_j $ for token $ \tau_k $
-    if the rate $ p_{j|k} $ is at most $ \pi $".
-  }
-\end{center}
-
-\vspace{-.4cm}
-Analogously, a limit sell order is represented by a tuple $ (j,k,+\infty,\overline{y},\pi) $
-meaning
-\vspace{-.6cm}
-\begin{center}
-  \emph{
-    "Sell (at most) $ \overline{y} $ units of token $ \tau_k $ for token $ \tau_j $
-    if the rate $ p_{k|j} $ is at least $ \frac{1}{\pi} $".
+    if the exchange rate $ p_{j|k} $ is at most $ \pi $".
   }
 \end{center}
-\vspace{-.3cm}
-
-Since we are only considering these two order types, either $ \overline{x} $ or $ \overline{y} $
-needs to be $ +\infty $, whereas the other needs to take a finite value.
-
-\newpage
-The semantics of the two order types are somewhat similar in that there is one token to be
-exchanged against another, if a certain limit price is guaranteed.
-Such orders are hence referred to as \emph{limit orders}.
+\vspace{-.5cm}
 The difference between buy and sell orders is the following:
 \begin{itemize}
-  \item In a buy order, a fixed (maximum) buy volume $ \overline{x} $ is set, so the buyer
+  \item In a buy order, a fixed (maximum) buy volume $ \overline{x} $ is set, so the trader
   indicates a preference to buy $ \overline{x} $ units of token $ \tau_j $.
   The amount of units of token $ \tau_k $ to be sold is flexible, as long as the limit price is
   respected (the smaller, the better).
-  \item In a sell order, a fixed (maximum) sell volume $ \overline{y} $ is set, so the seller
+  \item In a sell order, a fixed (maximum) sell volume $ \overline{y} $ is set, so the trader
   indicates a preference to sell $ \overline{y} $ units of token $ \tau_k $.
   The amount of units of token $ \tau_j $ to be bought is flexible, as long as the limit price
   is respected (the higher, the better).
 \end{itemize}
 \vspace{.3cm}
 
-Nevertheless, the question remains whether it makes sense to keep the distinction between buy and
-sell orders.
-Consider a buy order $ (j,k,\overline{x},+\infty,\pi) $, which reveals that the buyer would be
-ready to sell at most $ \pi \cdot \overline{x} $ tokens $ \tau_k $ in exchange for tokens
-$ \tau_j $.
-Then, from an economically rational point of view, the buyer should not object to receiving more
-than $ \overline{x} $ tokens $ \tau_j $ for a maximum of $ \pi \cdot \overline{x} $ tokens
-$ \tau_k $, \ie if the exchange rate $ p_{j|k} $ is lower than $ \pi $.
-Hence, the buy order could be translated into an equivalent sell order
-$ (j,k,+\infty,\pi \cdot \overline{x},\pi) $.
-In this paper, however, we will continue to treat the different order types separately.
-
+However, for notational simplicity, we will restrict ourselves to only considering sell orders in
+this paper.
+In fact, assuming that the trader has constant utility for an arbitrary number of buy-tokens,
```

Comment:
This is "constant utility" during the batch cycle, right?

Also constant risk preferences.